INSTALL.md

```diff
@@ -29,18 +29,7 @@ There are several other distributions of the PureScript compiler available, whic
 
 ## Compiling from source
 
-The easiest way is to use stack:
-
-```
-$ stack update
-$ stack unpack purescript
-$ cd purescript-x.y.z  # (replace x.y.z with whichever version you just downloaded)
-$ stack install --flag purescript:RELEASE
```

Comment:
Is the `purescript:RELEASE` flag still necessary? I removed those from the new instructions since omitting it wasn't causing issues for me.

Reply:
Can you please leave this file as-is? This file is really intended for people wanting to install the compiler, not to contribute to it. The RELEASE flag is needed for ensuring that the output of `purs --version` is correct.

Reply:
For users who just want to install the compiler (without contributing), would it be simpler to replace that section with:
```
stack install purescript
```
I'm having trouble envisioning what target audience would prefer the above instructions, since both of these approaches rely on trusting stack + hackage.

Reply:
No, `stack install purescript` would behave differently to what we have there; it tries to install it in a particular snapshot and doesn't use `purescript`'s own stack.yaml file.